Windows/Main.Designer.cs

```diff
@@ -308,6 +318,9 @@ private void InitializeComponent()
             this.objectListView.AllColumns.Add(this.olvColumnSignature);
             this.objectListView.AllColumns.Add(this.olvColumnPermission);
             this.objectListView.AllColumns.Add(this.olvColumnError);
+            this.objectListView.Anchor = ((System.Windows.Forms.AnchorStyles)((((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Bottom) 
```

Comment:
I think it was to be able to put the list -1 pixel left and 2 or 3 extra pixels to the right, removing the right extra space / border allows us to click on the extreme right of the screen to use the scroolbar when the window is maximized, the -1 pixel on left removes the border to so it matches the "design" and looks nice ^^

Reply:
Did not see your last sentence first, I can't see any difference when resizing between both versions, here's a capture https://youtu.be/oisT2CqDzZE
first window is no border, 2nd is latest 7.1 release

Reply:
Hi, thanks for making video for it. Very much appreciated 😃
Sorry I mispoke, I meant maximizing and restoring the window, although it may be because I'm using an older PC though so it's more noticeable

Reply:
No problem :)